cmd/snap/cmd_buy.go

```diff
@@ -117,33 +110,25 @@ Once completed, return here and run 'snap buy %s' again.`), snap.Name)
 		return err
 	}
 
-	reader := bufio.NewReader(nil)
-	reader.Reset(Stdin)
+	// TRANSLATORS: %q, %q and %s are the snap name, developer, and price. Please wrap the translation at 80 characters.
+	fmt.Fprintf(Stdout, i18n.G(`Please re-enter your Ubuntu One password to purchase %q from %q
```

Comment:
This is the exact wording in the UX document, so would prefer to keep as-is, if possible.